docs/index.md

```diff
@@ -0,0 +1,39 @@
+# Pollen Operator
+
+A Juju charm for deploying and managing a Pollen server.
+
+## Overview
+
```

Comment:
I'd suggest just removing all content above this - I don't think we need the `# Pollen Operator` header and content, or the `# Overview` header. The way things are displayed on charmhub this would mean a lot of whitespace before the useful content of the page, so let's just start directly with the content below (which looks great).

Reply:
(If we need a header for upload-charm-docs to work, then keeping `# Overview` would be okay).